packages/pigeon/lib/cpp_generator.dart

```diff
@@ -7,6 +7,9 @@ import 'functional.dart';
 import 'generator_tools.dart';
 import 'pigeon_lib.dart' show Error;
 
+/// Documentation open symbol.
+const String openDoc = '///';
```

Comment:
In all the files, these constants should either be `_`-prefixed, or annotated with `@visibleForTesting`, depending on whether you need them in tests.